test/cmd/basicresources.sh

```diff
@@ -236,6 +236,7 @@ os::cmd::expect_success_and_text "oc set env bc --all FOO-" "updated"
 os::cmd::expect_success "oc create secret generic mysecret --from-literal='foo.bar=secret'"
 os::cmd::expect_success_and_text "oc set env --from=secret/mysecret --prefix=PREFIX_ dc/test-deployment-config" "updated"
 os::cmd::expect_success_and_text "oc set env dc/test-deployment-config --list" "PREFIX_FOO_BAR="
+os::cmd::expect_success_and_text "oc set env dc/test-deployment-config --list --resolve" "PREFIX_FOO_BAR=secret"
```

Comment:
Another good test would be to run a resolve command as a user who only has the view role, when there are a lot of envvars referencing secrets, and make sure the output is not overwhelming, since every secret look up will be forbidden